File: abc/abc156/abc156_b.cs
// https://atcoder.jp/contests/abc156/submissions/10293058

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var input = Console.ReadLine().Split().Select(long.Parse).ToArray();
            var N = input[0];
            var K = input[1];

            var result = 0L;
            while (N > 0L)
            {
                N /= K;
                result++;
            }
            Console.WriteLine($"{result}");
        }
    }
}

File: abc/abc156/abc156_b.fs
// https://atcoder.jp/contests/abc156/submissions/11879844

let [| N; K |] = stdin.ReadLine().Split() |> Array.map int

let rec solver r n =
    match n with
    | n when n > 0 -> solver (r + 1) (n / K)
    | _ -> r

solver 0 N |> stdout.WriteLine
